src/prediction.jl

```diff
@@ -249,4 +249,68 @@ function build_future_df(base_df, to_predict::Symbol)
 
     return(reduce(vcat, result_lines))
 
-end # end function1
\ No newline at end of file
+end # end function1
+
+function tryparsecol(Type::DataType, col)
+    try
+        return(parse.(Type, col))
+    catch
+        return(col)
+    end
+end
+
+function univariate_tietjenmoore(values::Vector{T} where T <: Real, k::Int64; alpha = 0.05, sim_trials = 100000, return_indexes = true)
+
+    function compute_tietjenmoore(data, k, n)
```

Comment:
Do these functions need to be defined inside the other function? Doesn't this mess with optimizations?